paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -846,10 +847,12 @@ def autoscaling_is_paused():
         return False
 
 
-def is_autoscaling_resumed(formatted_application: Union[V1Deployment, V1StatefulSet]):
-    if formatted_application.metadata.annotations:
-        if "is_paused" in formatted_application.metadata.annotations:
-            return formatted_application.metadata.annotations["is_paused"] == "False"
+def is_autoscaling_resumed(service_config: KubernetesDeploymentConfig):
+    kube_client = KubeClient()
+    annotations = get_annotations_for_deployment(kube_client, service_config)
+    if annotations:
+        if "is_paused" in annotations:
```

Comment:
can you make this a tiny bit more verbose, how about `paasta_service_autoscaler_is_paused` ?

Reply:
Good point, to avoid confusion with `autoscaling_is_paused()` which reads pause time from zk, I've changed this to `is_deployment_marked_paused()`